scripts/gitlint/zephyr_commit_rules.py

```diff
@@ -104,7 +104,7 @@ class MaxLineLengthExceptions(LineRule):
     id = "UC4"
     target = CommitMessageBody
     options_spec = [IntOption('line-length', 80, "Max line length")]
-    violation_message = "Line exceeds max length ({0}>{1})"
+    violation_message = "Commit line exceeds max length ({0}>{1})"
```

Comment:
I'd add "body" here as well, what do you think? (thinking the actual git terminology should be "commit message" but I guess either works)

Reply:
Yes I can do that